overlord/snapstate/snapstate_update_test.go

```diff
@@ -13429,6 +13429,31 @@ func (s *snapmgrTestSuite) TestAutoRefreshSplitRefresh(c *C) {
 	c.Check(chg.Status(), Equals, state.DoneStatus)
 }
 
+func (s *snapmgrTestSuite) TestRefreshWithRegistry(c *C) {
```

Comment:
don't we need also a install with registries test?

Reply:
There was one in `target_test.go`. This was done before the refactor, so the install test covered the bit in `target.go` and the update test covered the other bit that has since been removed by merging with the refactor. I've now put them both in `snapstate_update_test.go`